conf/application.conf

```diff
@@ -18,8 +18,9 @@ appName = "digital-engagement-platform-skin"
 play.http.router = prod.Routes
 
 # Provides an implementation and configures all filters required by a Platform frontend microservice.
-play.filters.enabled += play.filters.csp.CSPFilter
-play.filters.csp.directives.script-src = "'self' 'unsafe-inline' localhost:9193 localhost:9032 localhost:12345 www.google-analytics.com www.googletagmanager.com *.optimizely.com *.digital.nuance.com data:"
+# play.filters.enabled += play.filters.csp.CSPFilter
+# play.filters.csp.directives.script-src = "'self' 'unsafe-inline' localhost:9193 localhost:9032 localhost:12345 www.google-analytics.com www.googletagmanager.com *.optimizely.com *.digital.nuance.com data:"
```

Comment:
Should not need both this and the new contentSecurityPolicy config, please remove